code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -269,6 +269,13 @@
 	category = "Defensive"
 	cost = 1
 
+/datum/spellbook_entry/honk
+	name = "Curse of the Clown"
+	spell_type = /obj/effect/proc_holder/spell/targeted/touch/honk
+	category = "Offensive"
+	cost = 3
+	no_coexistance_typecache = /obj/effect/proc_holder/spell/targeted/touch/flesh_to_stone // there may only be one transformative touch spell

```

Comment:
why tho

Reply:
bro I wanna do pimp hand wizard (Smite+Curse of the Clown+Flesh to Stone+some other shit idk)

how could you do this to me

Reply:
highlander rules

Reply:
then again this could be pure misery (clown statue, be unable to do anything for 5 minutes then be a clown for the rest of the shift until someone sprays you with acid)